ui/src/apps/clusterInfo/components/ClusterInfoTable.js

```diff
@@ -99,4 +122,23 @@ function wrapNode(node, comp, id) {
   }
 }
 
+async function deleteTiDBTopology(node, dataSource, setCluster) {
+  let resp = await client.dashboard.topologyTidbAddressDelete(node.address)
+  if (resp.status === 200) {
```

Comment:
I don't think we should judge `resp.status === 200`, it is already done in the client lib internal, you can consider using `try...catch` to handle the exception.